var/spack/repos/builtin/packages/libint-cp2k/package.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class LibintCp2k(CMakePackage):
+    """libint configured for CP2K."""
+
+    homepage = "https://github.com/cp2k/libint-cp2k"
+
+    version('2.6.0', sha256='1cd72206afddb232bcf2179c6229fbf6e42e4ba8440e701e6aa57ff1e871e9db')
+
+    depends_on('python', type='build')
+    depends_on('boost')
+
+    def url_for_version(self, version):
+        url = 'https://github.com/cp2k/libint-cp2k/releases/download/v{0}/libint-v{0}-cp2k-lmax-{1}.tgz'
+
+        return url.format(version, 5)
```

Comment:
What's up with the 5? Why use `.format` if it's always 5?

Reply:
I couldn't figure out yet how to do a variant dependent `version`. This tarball of libint is configured with a fixed maximum angular momentum, but depending on what the user of CP2K wants to do, there might be a case that someone would like to use a higher one.
Meaning, we provide prepared tarballs for `lmax` of 4,5,6,7.
So, ideally, (if we don't integrate this in the `libint` package directly) there is a variant `lmax` and depending on this selection, a different tarball is fetched and used.

Reply:
I see. This isn't currently supported, but it's something I would like to see. You could hack `url_for_version` to download the right tarball based on the variant, but there's currently no way to change the checksum based on variant.